src/index.html

```diff
@@ -1,18 +1,15 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Product cards</title>
-    <link
-      rel="stylesheet"
-      href="./styles/index.scss"
-    />
-  </head>
-  <body>
-    <h1>Product cards</h1>
-  </body>
-</html>
+<!DOCTYPE html>
+<div class="product-card" data-qa="card">
```

Comment:
The `product-card` class should be separated into its own BEM block file as per the task requirements. Consider creating a separate file for this block.